### PR TITLE
✨Add Rules page

### PR DESCRIPTION
**Pull request by fre-ben**
- Add Rules page
- Style Rules page
- Add Rules page to storybook

Deployment: https://sisu-pipelin-rulespage-jawwtma.herokuapp.com/rules
Part of issue #1

### CODE REVIEW

File: pages/rules.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import BackBtn from "../components/button/BackBtn";
+import Logo from "../components/logo/Logo";
+import styles from "../styles/Rules.module.css";
+
+export default function Rules() {
+  const router = useRouter();
+
+  const goHome = () => {
+    router.push("/");
+  };
```
Comment (phlgr) [inline pages/rules.tsx]: I would not use a function to just push the user to the homepage.
Use the Link Component from Next instead.

File: pages/rules.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      <Head>
+        <title>How to play</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <main>
+        <div className={styles.container}>
+          <Logo size="big" />
+          <div className={styles.menuItems}>
+            <BackBtn onClick={goHome} />
```
Comment (phlgr) [inline pages/rules.tsx]: https://nextjs.org/docs/api-reference/next/link

File: pages/rules.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+              jig carouser broadside cable strike colors. <br />
+              <br />
+              Bring a spring upon her cable holystone blow the man down spanker
+              Shiver me timbers to go on account lookout wherry doubloon chase.
+              Belay yo-ho-ho keelhaul squiffy black spot yardarm spyglass sheet
+              transom heave to. Trysail Sail ho Corsair red ensign hulk smartly
+              boom jib rum gangway. Case shot Shiver me timbers gangplank crack
+              Jennys tea cup ballast Blimey lee snow crows nest rutters. Fluke
+              jib scourge of the seven seas boatswain schooner gaff booty Jack
+              Tar transom spirits.
```
Comment (phlgr) [inline pages/rules.tsx]: Put this into a `<p>`-Tag and remove the `<br>`-Tags

File: styles/Rules.module.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
+.container {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
```
Comment (phlgr) [inline styles/Rules.module.css]: I would turn this into a component.
https://reactjs.org/docs/composition-vs-inheritance.html
Also be careful when using `vh` and `vw`

File: styles/Rules.module.css [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+.menuItems {
+  margin-top: 4em;
+  max-width: 90vw;
```
Comment (phlgr) [inline styles/Rules.module.css]:
```suggestion
  max-width: 90%;
```

File: styles/Rules.module.css [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  display: grid;
+  grid-auto-flow: column;
+  row-gap: 1.5em;
```
Comment (phlgr) [inline styles/Rules.module.css]:
```suggestion
```

File: styles/Rules.module.css [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+  align-items: start;
+  justify-items: center;
```
Comment (phlgr) [inline styles/Rules.module.css]:
```suggestion
```

File: styles/Rules.module.css [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+  column-gap: 3.5em;
+}
+
+.rules {
+  font-size: 1.5em;
+  max-width: 40rem;
+  overflow-y: auto;
+  max-height: 60vh;
```
Comment (phlgr) [inline styles/Rules.module.css]: You should look for a way to remove these properties, maybe even try to put everything into one single grid.

**Review by phlgr — CHANGES_REQUESTED** [review]
GJ 🚀 
Just a few quick fixes 😄

File: styles/Rules.module.css (hunk 4 above)
Comment (fre-ben) [reply styles/Rules.module.css]: Going to do this in another PR, thanks!

**Pull request merged**